.git-hooks/pre-commit

```diff
@@ -129,6 +129,17 @@ sub check_author()
     }
 }
 
+sub check_prettier()
+{
+    my $prettier = `make prettier`;
```

Comment:
This might not work as expected. It would be better to add a system call here.

For example: ```system(`make prettier`)```